### PR TITLE
Update Quien es Quien API to V3

### LINKED ISSUE #735: Change QuienEsQuien.Wiki API Link Documentation
The one that is in the spider is obsolete, need to be changed to 
https://qqwapi-elastic.readthedocs.io/es/latest/

### PR DESCRIPTION
**Pull request by cecicasco**
Closes #735

### CODE REVIEW

**Comment by yolile** [comment]
Hmm, https://api.quienesquien.wiki/v3/contracts returns single releases now, but it has sorting eg https://api.quienesquien.wiki/v3/contracts?sort=date&sort_direction=desc, but they also publish https://api.quienesquien.wiki/v3/record that returns record packages, but that service doesn't implement sorting. For incremental updates for the data registry, I guess we prefer an endpoint with sorting, but we will miss the package metadata. @jpmckinney should we implement only the contracts/releases endpoint or should we have two, one for releases and another one for record packages?

File: kingfisher_scrapy/spiders/mexico_quien_es_quien.py [hunk 1]
```diff
@@ -23,15 +23,15 @@ class MexicoQuienEsQuien(IndexSpider):
     data_type = 'record_package'
```
Comment (yolile) [inline kingfisher_scrapy/spiders/mexico_quien_es_quien.py:23]:
```suggestion
    data_type = 'release'
```

File: kingfisher_scrapy/spiders/mexico_quien_es_quien.py [hunk 2]
```diff
@@ -23,15 +23,15 @@ class MexicoQuienEsQuien(IndexSpider):
...
 
     # IndexSpider
-    count_pointer = '/data/0/collections/contracts/count'
+    count_pointer = '/data/index/contracts/count'
     limit = 1000
-    base_url = 'https://api.quienesquien.wiki/v2/contracts?sort=-compiledRelease.date'
+    base_url = 'https://api.quienesquien.wiki/v3/contracts'
```
Comment (yolile) [inline kingfisher_scrapy/spiders/mexico_quien_es_quien.py:28]:
```suggestion
    base_url = 'https://api.quienesquien.wiki/v3/contracts?sort=date&sort_direction=desc'
```

**Review by yolile — APPROVED** [review]

**Comment by jpmckinney** [comment]
> Hmm, https://api.quienesquien.wiki/v3/contracts returns single releases now, but it has sorting eg https://api.quienesquien.wiki/v3/contracts?sort=date&sort_direction=desc, but they also publish https://api.quienesquien.wiki/v3/record that returns record packages, but that service doesn't implement sorting. For incremental updates for the data registry, I guess we prefer an endpoint with sorting, but we will miss the package metadata. @jpmckinney should we implement only the contracts/releases endpoint or should we have two, one for releases and another one for record packages?

We can create an issue to support the other endpoint. In other cases, we have spiders for each endpoint (like Afghanistan).

**Comment by jpmckinney** [comment]
Aha, I see #739.

**Pull request merged**